### PR TITLE
Happychat Support Page

### PR DESCRIPTION
**Pull request by beaucollins**
Adds a happychat support page with a full page chat client.

Mobile clients are sent to this page instead of using the docked sidebar.

Depends on:

- #7758 
- #7760 

For more context see #7757 

Test live: https://calypso.live/?branch=add/happychat-page

### CODE REVIEW

File: client/wordpress-com.js [hunk 1]
```diff
@@ -67,6 +67,13 @@ sections = [
 		secondary: true
 	},
 	{
+		name: 'chat',
```
Comment (mjangda) [inline client/wordpress-com.js]: We should move this outside of the `sections` literal and wrap inside an `isEnabled` check further down the page.

**Comment by mjangda** [comment]
🎈 🎂  🎉

File: client/me/sidebar/index.jsx [hunk 2]
```diff
@@ -160,6 +161,15 @@ const MeSidebar = React.createClass( {
 							onNavigate={ this.onNavigate }
 							preloadSectionName="help"
 						/>
+						{ config.isEnabled( 'happychat' )
+							? <SidebarItem
+								selected= { selected === 'happychat' }
+								link="/me/chat"
+								icon="comment"
+								label= { this.translate( 'Support Chat' ) }
+								onNavigate={ this.onNavigate } />
```
Comment (mtias) [inline client/me/sidebar/index.jsx]: It'd be good to add preloading here.

File: client/me/chat/index.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+*/
+import React from 'react';
+import page from 'page';
+
+/*
+	Internal deps
+*/
+import { renderWithReduxStore } from 'lib/react-helpers';
+import controller from 'me/controller';
+import LiveChat from 'components/happychat/page';
```
Comment (mtias) [inline client/me/chat/index.jsx]: This `page` subcomponent should be moved to this folder (`/me/chat/`) as `main.jsx`.

File: client/me/sidebar/index.jsx (hunk 2 above)
Comment (beaucollins) [inline client/me/sidebar/index.jsx]: see #7788

File: client/me/chat/index.jsx (hunk 3 above)
Comment (beaucollins) [inline client/me/chat/index.jsx]: see #7788

**Pull request merged**